src/Components/Main.js

```diff
@@ -0,0 +1,186 @@
+import React,{useState,useEffect} from 'react';
+import { BrowserRouter as Router, Routes,Route,Link} from 'react-router-dom';
+import { useNavigate } from 'react-router-dom';
+import {auth} from "../firebase"
+import { getAuth, RecaptchaVerifier ,signInWithPhoneNumber} from 'firebase/auth';
+import {  logout,login } from '../features/userSlice';
+import { useDispatch,useSelector } from 'react-redux';
+import userData from "../UserData.json"
+import tick from "../assets/tick.png"
+import Otp from './Otp';
+import Form from './Form';
+
+
+
+const Mainpage = () => {
+    const [mobile, setMobile] = useState("")
+    let userArray=userData;
+    const otpauth= getAuth();
+    const [verify,setVerify]=useState(false)
+    
+    const [viewOtpForm, setViewOtpForm] = useState(false);
+    const navigate= useNavigate();
+    const name=JSON.parse(localStorage.getItem("user")).name
```

Comment:
This can be an object called userData with name, email and uid, doesn't have to be individual variables